microservices/frame_engine/cameraPi.py

```diff
@@ -0,0 +1,26 @@
+import time
+import picamera
+import numpy as np
+import cv2
+
+from config import *
+
+# initialize the camera and grab a reference to the raw camera capture
+camera = PiCamera()
+rawCapture = PiRGBArray(camera)
+# allow the camera to warmup
+time.sleep(0.1)
+# grab an image from the camera
+camera.capture(rawCapture, format="bgr")
+image = rawCapture.array
+# display the image on screen and wait for a keypress
+cv2.imshow("Image", image)
+cv2.waitKey(0)
+
+# with picamera.PiCamera() as camera:
```

Comment:
Is this code in here still needed?

Reply:
Removed for now, but i will need to test this approach when a get the camera.